src/NServiceBus.Transport.SQS/IAmazonSqsIncomingMessageExtractor.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NServiceBus.Transport.SQS
+{
+    using System.Collections.Generic;
+    using Amazon.SQS.Model;
+
+    /// <summary>
+    /// Configures how the incoming SQS transport message is extracted
+    /// </summary>
+    public interface IAmazonSqsIncomingMessageExtractor
```

Comment:
As discussed today, we can rename this removing the reference to AmazonSQS. For example, `IIncomingMessageExtractor` or an even simpler `IMessageExtractor`.

Reply:
Changed to IMessageExtractor.